### DISCUSSION TITLE
articles/poisson-distribution

### DISCUSSION THREAD

**Discussion by giscus** [mirsazzathossain/mirsazzathossain.me#240] [General]
# articles/poisson-distribution

The blog post explores the Poisson distribution, a statistical distribution commonly used in various fields, including astronomy, to model random events. It explains the properties of the distribution, its real-world applications, and provides a step-by-step guide on how to visualize the Poisson distribution using Python. The post also discusses the issue of Poisson noise in astronomical observations and presents a practical example of how to calculate and deal with it using Python.

https://www.mirsazzathossain.me/articles/poisson-distribution

**Comment by lucasjinreal** [comment]
rth
